commit 5feb09266fd2928ec0a4dcb98c1dc7f053111316
Author: Jia Tan <jiat0218@gmail.com>
Date:   Wed Dec 20 22:43:44 2023 +0800

    Build: Allow sandbox to be configured for just xzdec.
    
    If xz is disabled, then xzdec can still use the sandbox.

diff --git a/configure.ac b/configure.ac
index cdd72a9f..a4ef57a5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -523,15 +523,15 @@ AC_ARG_ENABLE([sandbox], [AS_HELP_STRING([--enable-sandbox=METHOD],
 		The default is 'auto' which enables sandboxing if
 		a supported sandboxing method is found.])],
 	[], [enable_sandbox=auto])
-case $enable_xz-$enable_sandbox in
-	no-*)
+case $enable_xzdec-$enable_xz-$enable_sandbox in
+	no-no-*)
 		enable_sandbox=no
-		AC_MSG_RESULT([no, --disable-xz was used])
+		AC_MSG_RESULT([no, --disable-xz and --disable-xzdec was used])
 		;;
-	*-auto)
+	*-*-auto)
 		AC_MSG_RESULT([maybe (autodetect)])
 		;;
-	*-no | *-capsicum | *-pledge | *-landlock)
+	*-*-no | *-*-capsicum | *-*-pledge | *-*-landlock)
 		AC_MSG_RESULT([$enable_sandbox])
 		;;
 	*)